src/get-default-auth-cfg.js

```diff
@@ -9,7 +9,7 @@ module.exports = async () => {
   let migrated = false
 
   const config = {
-    _: 'This is your Now credentials file. DON\'T SHARE! More: https://git.io/v5ECz',
+    _: 'This is your Now credentials file. DON\'T SHARE! More: https://zeit.co/docs/features/configuration#now.json',
```

Comment:
This actually isn't correct. The link you are updating it to is about your project local `now.json` file, _not_ the `~/.now/auth.json` file.

Reply:
I see. I searched the README history to find an anchor for ```global-configuration``` (where the git.io leads to) to no avail. Maybe you can point to the right place?

Reply:
https://goo.gl/mbf4CZ